gitman/models/config.py

```diff
@@ -29,22 +30,45 @@ def __init__(self, root, filename="gitman.yml", location="gitman_sources"):
         self.location = location
         self.sources = []
         self.sources_locked = []
+        self.path = self.config_path
 
     @property
     def config_path(self):
         """Get the full path to the configuration file."""
         return os.path.normpath(os.path.join(self.root, self.filename))
-    path = config_path
+
+    @config_path.setter
+    def config_path(self, value):
+        """Set the full path to the configuration file."""
+        self._config_path = os.path.normpath(value)
+
+    @property
+    def path(self):
+        return os.path.normpath(self._path)
+
+    @path.setter
+    def path(self, value):
+        self._path = os.path.normpath(value)
 
     @property
     def log_path(self):
         """Get the full path to the log file."""
-        return os.path.join(self.location_path, self.LOG)
+        return os.path.normpath(os.path.join(self.location_path, self.LOG))
+
+    @log_path.setter
+    def log_path(self, value):
+        """Set the full path to the log file."""
+        self._log_path = os.path.normpath(value)
 
     @property
     def location_path(self):
-        """Get the full path to the dependency storage location."""
-        return os.path.join(self.root, self.location)
+        """Get the full path to the sources location."""
+        return os.path.normpath(os.path.join(self.root, self.location))
+
+    @location_path.setter
+    def location_path(self, value):
+        """Set the full path to the sources location."""
+        self._location_path = os.path.normpath(value)
```

Comment:
Similar comment to above, `self._location_path` doesn't seem to be used anywhere.